### PR TITLE
RevisedDBpediaQA

### PR DESCRIPTION
**Pull request by Sara-Farazi**
This is the script that creates the revised datasets. For each query, first I get the list of all (is dbo:wikiPageRedirects of) attribute and add them to the subject list. Then I query for objects for each (subject and predicate) pair, and add them as the ObjectList to the each query. The ObjectList is an empty list for a small number of queries, due to the disambiguation problem. (5 queries in valid.json, 7 queries in test.json and 26 queries in train.json). The script is in "revise_data_script" folder.
New datasets are very large, so I uploaded them in google drive. The link is in the README.md in the same folder.

### CODE REVIEW

**Comment by lintool** [comment]
@Sara-Farazi How big are the expanded datasets? If less than 100 MB we can check in directly. Should we call this v1.1?

**Comment by MichaelAzmy** [comment]
@lintool I just checked the files. The training set alone is 2.3 GB :D

**Comment by Sara-Farazi** [comment]
Yes, the train data is very large. Test data and valid data are smaller, but they are not less than 100 MB either!

**Comment by lintool** [comment]
Can you explain why the dataset increased in size so much?

**Comment by Sara-Farazi** [comment]
@lintool We are adding all the objects found for each pair of (subject, predicate). For some cases (e.g. birthplace, deathplace, etc) there are a lot of objects for each subject. That's why the the dataset is so large now.

**Comment by Impavidity** [comment]
@MichaelAzmy Do we necessary need to keep all the object there ?

File: revise_data_script/add_redirets_objects.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+	# if there is (") in the subject, fix it with urlencoder. This is the only case that needs urlencode.
+	if ('"' in subject):
+		subject = make_subject(subject)
+
+	q = "SELECT" + " *" + " {" + " ?redirects"  + " <http://dbpedia.org/ontology/wikiPageRedirects> <{}> ".format(subject) + "}"
+	sparql = SPARQLWrapper("http://dbpedia.org/sparql")
+	sparql.setQuery(q)
+	sparql.setReturnFormat(JSON)
+	results = sparql.query().convert()
+
```
Comment (Impavidity) [inline revise_data_script/add_redirets_objects.py]: remove consecutive empty line

Comment (Impavidity) [reply revise_data_script/add_redirets_objects.py]: The indent size is weird in this file. Could we use 4 spaces for indentation ?

**Comment by MichaelAzmy** [comment]
Maybe we can split the files into different parts. Questions and subjects in a file and objects in another file. What do you think?

**Pull request merged**